src/Feature.h

```diff
@@ -23,6 +23,12 @@ class Feature
   static const Feature ExperimentalPythonEngine;
 #endif
 
+#ifdef ENABLE_GUI_TESTS
+  static const bool HasGuiTesting {true};
+#else
+  static const bool HasGuiTesting {false};
```

Comment:
Might as well use constexpr here, to make sure it's guaranteed to be defined at compile-time. There are some rather complex C++ rules for when `static const` resolves at compile time and it changes based on how the values are used, so better be safe I think and declare them as constexpr.